docs/en/userguide/field-types.md

```diff
@@ -52,17 +52,14 @@ automatically when a form is filled out.
 Selecting a File Upload Field adds a field where users can upload a file from their
 computers. This is useful for getting documents and media files.
 
-The folder that this field uploads to can be customised by selecting "Show Options"
-and then selecting a new folder from the "Select upload folder" option. If no folder
-is selected it will upload by default to the "Uploads" folder.
+You will be prompted to choose a folder that this field uploads files to. 
+By default the "Form-Submissions" folder will be selected, however you’re free to create your own folder 
+(Form-Submissions/<your-submission-folder>) or choose an existing folder.
```

Comment:
Just make sure that `Form-Submissions/<your-submission-folder>` renders correctly after deploy.

Reply:
Styling doesn't output it 🙁

Reply:
Guess I could just put Form-Submissions/your-submission-folder